wasm/src/lib.rs

```diff
@@ -155,7 +155,12 @@ where
     }
 }
 
-fn run_internal<F>(code: &str, expr: &str, event_cb: F, shots: u32) -> Result<(), stateful::Error>
+fn run_internal<F>(
+    code: &str,
+    expr: &str,
+    event_cb: F,
+    shots: u32,
+) -> Result<(), Box<stateful::Error>>
```

Comment:
Why did this end up needing to be boxed?

Reply:
I started getting a warning that the Err part of this Result was too large. I'm unsure why I was getting that waring considering it was also telling me the size hadn't change since before my changes, _and_ `String` objects are smaller than `Ty` objects. In any case, it was directing me to box the Err, so I did to alleviate the warning.

Reply:
I see the same thing on my end... with the changes or without them, `stateful::Error` is 128 bytes, but only with the changes does clippy complain about the size. Very odd, but if a `Box` at the top level is enough to satisfy it then I suppose it's not a big concern.

Reply:
Wow, that is a large error type... I guess we normally store these in a `Vec<>` instead of returning directly, so they're already heap-allocated in other places. Clippy's fickleness aside, the reasoning makes sense to me.